SU2_CFD/src/variables/CVariable.cpp

```diff
@@ -72,6 +72,14 @@ CVariable::CVariable(unsigned long npoint, unsigned long ndim, unsigned long nva
       AD_InputIndex.resize(nPoint,nVar) = -1;
       AD_OutputIndex.resize(nPoint,nVar) = -1;
     }
+
+    if (config->GetKind_Streamwise_Periodic() == ENUM_STREAMWISE_PERIODIC::MASSFLOW) {
+      SolutionExtra.resize(1) = su2double(1e-16);
+      if (config->GetMultizone_Problem()) {
+        SolutionExtra_BGS_k.resize(1) = su2double(1e-16);
+        ExternalExtra.resize(1) = su2double(0.0);
+      }
+    }
```

Comment:
This is hardcoded for now, or for longer as I currently am aware of more possible extra solutions (aoa in fixed cl and pressure-drop for massflow outlets or streamwise periodic). I wonder whether it makes sense to make this already more flexible at the moment?

Reply:
Otherwise this PR should be complete in terms of added things